lightning/restore/restore.go

```diff
@@ -474,6 +472,66 @@ func (rc *RestoreController) runPeriodicActions(ctx context.Context, stop <-chan
 	}
 }
 
+type gcLifeTimeManager struct {
+	runningJobsLock *sync.Mutex
+	runningJobs     *int32
+	oriGCLifeTime   *string
+}
+
+func newManager() gcLifeTimeManager {
```

Comment:
(and return a pointer here)

```suggestion
func newManager() *gcLifeTimeManager {
```